include/circt/Dialect/OM/OMAttributes.td

```diff
@@ -34,4 +34,22 @@ def ReferenceAttr : AttrDef<OMDialect, "Reference", [TypedAttrInterface]> {
   }];
 }
 
+def SymRefAttr : AttrDef<OMDialect, "SymRef", [TypedAttrInterface]> {
```

Comment:
Maybe it is worth writing out the4 name `SymbolRefAttr` and `"SymbolRef"`? I don't know... I wouldn't go so far as to write out `SymbolRefereceAttribute`, so I don't know where to draw the line with abbreviation... naming is hard. I guess upstream calls it `SymbolRef`, which is why I was thinking that.

Reply:
makes sense, but `SymbolRefAttr` conflicts with the `mlir::SymbolRefAttr`, so I prefixed it to  `om::OMSymbolRefAttr`.